src/renderer/components/screens/WelcomeScreen/OnboardingScreen.tsx

```diff
@@ -0,0 +1,116 @@
+import { DialogBody, DialogContent, DialogHeader } from '../../Dialog'
+import { getLogger } from '../../../../shared/logger'
+
+import styles from './styles.module.scss'
+
+type Props = {
+  selectedAccountId: number
+  onUnSelectAccount: () => Promise<void>
+  onExitWelcomeScreen: () => Promise<void>
+  onNextStep: () => void
+}
+
+const log = getLogger('renderer/components/OnboardingScreen')
+
+export default function OnboardingScreen(props: Props) {
+  const tx = useTranslationFunction()
+  const { openDialog } = useDialog()
+  const [showBackButton, setShowBackButton] = useState(false)
+
+  const onAlreadyHaveAccount = () => {
+    openDialog(AlternativeSetupsDialog)
+  }
+
+  const onScanQRCode = () => {
+    openDialog(ImportQrCode, { subtitle: tx('qrscan_hint') })
+  }
+
+  useEffect(() => {
+    // Show back button when user has already created and configured accounts.
+    // On a fresh DC start we will not have any yet.
+    const checkAccounts = async () => {
+      const accounts = await BackendRemote.listAccounts()
+      if (accounts && accounts.length > 1) {
+        setShowBackButton(true)
+      }
+    }
+
+    checkAccounts()
+  }, [])
+
+  const onClickBackButton = async () => {
+    try {
+      const acInfo = await BackendRemote.rpc.getAccountInfo(
+        props.selectedAccountId
+      )
+      if (acInfo.kind === 'Unconfigured') {
+        await props.onUnSelectAccount()
+        await EffectfulBackendActions.removeAccount(props.selectedAccountId)
+      }
+
+      props.onExitWelcomeScreen()
+    } catch (error) {
+      if (error instanceof Error) {
+        openDialog(AlertDialog, {
+          message: error?.message,
+          cb: () => {},
+        })
+      } else {
+        log.error('unexpected error type', error)
+        throw error
+      }
+    }
+  }
+
+  return (
+    <>
+      <DialogHeader
+        onClickBack={showBackButton ? onClickBackButton : undefined}
+        title='Welcome!'
+      />
+      <DialogBody>
+        <DialogContent>
+          <img
+            className={styles.welcomeScreenImage}
+            src='../images/intro1.png'
+          />
+          <p className={styles.welcomeScreenTitle}>
+            {tx('welcome_chat_over_email')}
+          </p>
+          <div className={styles.welcomeScreenButtonGroup}>
+            <Button
+              className={styles.welcomeScreenButton}
+              type='primary'
+              onClick={props.onNextStep}
+            >
+              Let's get started!
+            </Button>
+            <Button
+              className={styles.welcomeScreenButton}
+              type='secondary'
+              onClick={onScanQRCode}
+            >
+              Scan QR code
+            </Button>
+            <Button
+              className={styles.welcomeScreenButton}
+              type='secondary'
+              onClick={onAlreadyHaveAccount}
+            >
+              I already have an account
```

Comment:
Why not the more straight-forward "create account"/"sign up" and "log in". I know under "log in" there is also "add as second device" and "restore from backup", but that is still "logging in", but with extra stuff.

Reply:
let's leave wording/flow discussions outside this PR for now, even if things are not 100% perfect, we need to move forward